go.mod

```diff
@@ -3,16 +3,22 @@ module go.dedis.ch/dela
 go 1.13
 
 require (
-	github.com/golang/protobuf v1.3.4
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
```

Comment:
I think those indirect imports will disappear with a go mod tidy

Reply:
It's actually the `go mod tidy` that made them appear 🤔